vars/mtaBuild.groovy

```diff
@@ -65,8 +67,16 @@ def call(Map parameters = [:]) {
             }
         }
 
-        def mtaYaml = readYaml file: "${pwd()}/mta.yaml"
+        def mtaYmlName = "${pwd()}/mta.yaml"
+        def applicationName = configuration.applicationName
+
+        if (!fileExists(mtaYmlName) && applicationName) {
```

Comment:
What happens in case there is no mtaYml file and no application name? Should we fail in this case here?

Reply:
I would not change the logic of the mtaBuild here.
Before introducing the applicationName, if the mta.yml file doesn't exists, the MtaBuild throws FileNotFoundException.
I added an echo with the information that the applicationName is not set - the user could then check the documentation and decide on providing it or checking-it the mta.yml